src/torrent.cpp

```diff
@@ -10558,22 +10558,23 @@ namespace libtorrent
 
 		// activity time
 #ifndef TORRENT_NO_DEPRECATE
-		st->finished_time = finished_time();
-		st->active_time = active_time();
-		st->seeding_time = seeding_time();
+		// cast to seconds in case that internal values doesn't have ratio<1>
+		st->finished_time = int(duration_cast<seconds>(finished_time()).count());
```

Comment:
if you duration_cast this to the type you defined above, ``seconds_t``, you don't need to wrap it in a cast to ``int``